### DISCUSSION TITLE
Basic Economy Program

### DISCUSSION THREAD

**Discussion by Skigamer13** [Rapptz/discord.py#6812] [General]
Does anybody know of a basic beginner-friendly starter economy code that is easy to customize and add commands to that works with Repl.it?

**Comment by Rapptz** [comment]
As mentioned in the previous discussion, this isn't really the place to ask for code or broad questions like these.

**Reply by Skigamer13** [reply]
oh sorry, my bad. the previous discussion just disappeared overnight for me so I thought I should try again incase there was an error.

**Reply by Skigamer13** [reply]
would it fit under the general category instead?

**Reply by Rapptz** [reply]
Nope.
